gif_sizer/parser.py

```diff
@@ -0,0 +1,14 @@
+import click
+
+@click.command(help='Resize an animated GIF')
+@click.argument('command')
+@click.option('--verbose', '-v', help='Enable verbose output', is_flag=True)
+@click.option('--downscale', type=bool, help='Downscale GIF format')
+def parse_command(command, verbose, downscale) -> dict:
```

Comment:
Well, It's obvious where all those parameters coming from.. but it still would be nice to add type hints..

Reply:
totally agree 👍 I should mark this PR as a draft, because I haven't finished yet 😉